Directory.Build.props

```diff
@@ -1,27 +1,32 @@
 <Project>
-  <PropertyGroup Label="Package information">
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <PackageProjectUrl>https://github.com/IntelliTect/Multitool</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/IntelliTect/Multitool</RepositoryUrl>
-    <Authors>IntelliTect</Authors>
-    <!-- Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element) -->
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+	<PropertyGroup Label="Package information">
 
-    <!-- Embed source files that are not tracked by the source control manager in the PDB -->
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+		<LangVersion>10.0</LangVersion>
+		<ImplicitUsings>true</ImplicitUsings>
+		<Nullable>enable</Nullable>
```

Comment:
This property group is named Package information, I would not expect to find properties like this within it. I would move these into their own property group (or into the Directory.Build.props file)